easybuild/easyconfigs/m/MotionCor2/MotionCor2-1.4.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,46 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Ake Sandgren
+# HPC2N
+# Umea University
+
+name = 'MotionCor2'
+version = '1.4.4'
+
+homepage = 'https://msg.ucsf.edu/'
+description = """MotionCor2 correct anisotropic image motion at the
+single pixel level across the whole frame, suitable for both single
+particle and tomographic images. Iterative, patch-based motion detection
+is combined with spatial and temporal constraints and dose weighting.
+
+Cite publication: Shawn Q. Zheng, Eugene Palovcak, Jean-Paul Armache,
+Yifan Cheng and David A. Agard (2016) Anisotropic Correction of
+Beam-induced Motion for Improved Single-particle Electron
+Cryo-microscopy, Nature Methods, submitted.
+BioArxiv: https://biorxiv.org/content/early/2016/07/04/061960
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+sources = [{
+    'filename': '%(name)s_%(version)s.zip',
+    'extract_cmd': 'unzip -j %s'
+}]
+checksums = ['128f5cdf83b5023204c10b4801baa13f0a56a950e0bc6e0ed26c7f3aa1df59d7']
+
+# No longer directly downloadable, available from https://msg.ucsf.edu/software
+# CUDA is a build dependency to make sure it gets installed.
+# It's actually a runtime dependency, but that's handled in the wrapper to
+# make sure it doesn't conflict with whatever toolchain happens to be loaded.
+# Change CUDA version to match the latest one used in this version
+# of MotionCor2
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CUDAcore', '11.1.1', '', True),
```

Comment:
And since this motioncor2 version has CUDA 11.3. has its highest cuda version, this should be changed to
('CUDA', '11.3.1', '', True)
since we should no longer use CUDAcore

Reply:
@akesandgren @smoors I changed sources and CUDA build dep.. I assume it is not required to have -CUDA-11.3.1 as version suffice, because it is  not a dependency, but a builddependency

Reply:
Technically speaking it is a runtime dep, but that is handled in the wrapper.
But no, we do not need the -CUDA suffix here.